docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,25 @@
 Version history
 ===============
 
+5.3.0-m2 (2016 November 23)
+---------------------------
+
+Fixes and clean up:
+
+* clean up of bundles, packages and build targets to reflect decoupling work
+* POM repositories clean up
+* updated developer documentation on building Bio-Formats
+* updated testing packages
+* updated build versioning from Maven
```

Comment:
I think we could expand on this to explain that 1- we unified the versioning strategy, 2- we reviewed the meta information stored in the manifest of each JAR and 3- we are now making introspecting this meta-information in the `FormatTools` API to retrieve the version and revision number. Does that correctly represent your contribution @rleigh-codelibre ?